scripts/generate-configs.ts

```diff
@@ -78,42 +88,30 @@ async function generateConfigs(): Promise<void> {
 
     return Object.fromEntries(
       filteredRules
-        .filter(ruleId => !DISABLED_RULES_FOR_ALL_CONFIG.has(ruleId))
+        .filter(ruleId => !rules[ruleId].meta.docs.isDisabledForAllConfig)
         .map(ruleId => [`@graphql-eslint/${ruleId}`, getRuleOptions(ruleId, rules[ruleId])])
     );
   };
 
-  writeFormattedFile(
-    'configs/schema-recommended.ts',
-    `export default ${JSON.stringify({
-      extends: ['plugin:@graphql-eslint/base'],
-      rules: getRulesConfig('Schema', true),
-    })}`
-  );
-
-  writeFormattedFile(
-    'configs/operations-recommended.ts',
-    `export default ${JSON.stringify({
-      extends: ['plugin:@graphql-eslint/base'],
-      rules: getRulesConfig('Operations', true),
-    })}`
-  );
-
-  writeFormattedFile(
-    'configs/schema-all.ts',
-    `export default ${JSON.stringify({
-      extends: ['plugin:@graphql-eslint/base', 'plugin:@graphql-eslint/schema-recommended'],
-      rules: getRulesConfig('Schema', false),
-    })}`
-  );
-
-  writeFormattedFile(
-    'configs/operations-all.ts',
-    `export default ${JSON.stringify({
-      extends: ['plugin:@graphql-eslint/base', 'plugin:@graphql-eslint/operations-recommended'],
-      rules: getRulesConfig('Operations', false),
-    })}`
-  );
+  writeFormattedFile('configs/schema-recommended.json', {
+    extends: './base.json',
+    rules: getRulesConfig('Schema', true),
+  });
+
+  writeFormattedFile('configs/operations-recommended.json', {
```

Comment:
the main purpose of this is to have configs in separate places instead of having them in `index.js` file.
So this will be easy to look up for debugging purposes.
![image](https://user-images.githubusercontent.com/7361780/146544430-21d378a0-f10e-4b43-aebf-0afa43c80ec5.png)